index.src.html

```diff
@@ -81,12 +85,6 @@
         text: name; url: attr-fe-autocomplete-name
         text: photo; url: attr-fe-autocomplete-photo
         text: username; url: attr-fe-autocomplete-username
-spec: RFC6454; urlPrefix: https://tools.ietf.org/html/rfc6454
-  type: dfn
-    text: the same; url: section-5
-    text: origin; url: section-3.2
-    text: ASCII serialization of an origin; url: section-6.2
-    text: ASCII serialization; url: section-6.2
```

Comment:
Thanks for reminding me. I ran bikeshed update, and also pulled the newest repository revision for bikeshed (without that, bikeshed ended up with a Python error of too many recursive calls).